glambie/util/unit_conversion_helpers.py

```diff
@@ -1,186 +1,227 @@
-def meters_to_gigatonnes(meters_list: list, rgi_area_km2: float, density_of_ice_in_gt_per_m3: float = 850) -> list:
+from typing import Iterable
```

Comment:
Module should be renamed to mass_height_conversions.py (clearly summarises what it's doing)